src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
Would either need full load operation or divide by RTF to get back there. Dividing by RTF is tricky since RTF is also modified by defrost operation. I'd have to see both versions.